script/test

```diff
@@ -1,3 +1,16 @@
-#!/bin/bash
-set -ex
-npm run check
+#!/usr/bin/env node
```

Comment:
Running `script/setup` should install all dependencies, I'm assuming it's already been run before this would be run.